extensions/converters/json/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~  Copyright 2019 Red Hat, Inc.
+ ~
+ ~  Licensed under the Apache License, Version 2.0 (the "License");
+ ~  you may not use this file except in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing, software
+ ~  distributed under the License is distributed on an "AS IS" BASIS,
+ ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~  See the License for the specific language governing permissions and
+ ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>io.smallrye.ext</groupId>
+        <artifactId>smallrye-config-extensions-converters</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>configconverters-json</artifactId>
```

Comment:
Would we need the spec version even in an extension ? As this is not implementing a spec ? Happy to do it, just checking.

Reply:
I've been debating this between myself today.

Though the extensions are less likely to be broken by Config spec changes, there is always the chance of a breaking change in the SPI.

So I think for simplicity it makes sense for the extensions to have the spec in the artifact id so they can be released at the same time as the rest of SmallRye Config. Otherwise it needs to be separate repos and we get into issues of not knowing if an extension is broken by a SmallRye Config change until we update to a new release.

Reply:
Ok cool. Will do.

Reply:
Done